pkg/storage/mysql/mysql.go

```diff
@@ -43,6 +43,7 @@ type Datastore struct {
 	dbStatsCollector       prometheus.Collector
 	maxTuplesPerWriteField int
 	maxTypesPerModelField  int
+	versionReady           storage.ReadinessStatus
```

Comment:
🤔 why store the whole struct if this is just relevant to version. 

Also, if you are going to keep this at the struct level, you can have a single method of `isReady` as not to continue adding to the interface. And just pass in if it needs to check the version.

Reply:
Changed to just keep bool